### PR TITLE
update for 0.7

### PR DESCRIPTION
**Pull request by simonbyrne**

### CODE REVIEW

File: src/DecFP.jl [hunk 1]
```diff
@@ -20,6 +20,10 @@ using Compat, Compat.Printf, Compat.Unicode
     export GC
 end
 
+@static if VERSION >= v"0.7.0-alpha.69"
+    import SpecialFunctions
```
Comment (stevengj) [inline src/DecFP.jl]: Doesn't this need to be in REQUIRE?

File: .travis.yml [hunk 2]
```diff
@@ -6,6 +6,7 @@ os:
 
 julia:
   - 0.6
+  - 0.7
```
Comment (quinnj) [inline .travis.yml]: Did this start working as of alpha/beta?

File: appveyor.yml [hunk 3]
```diff
@@ -1,9 +1,18 @@
 environment:
   matrix:
-  - JULIA_URL: "https://julialang-s3.julialang.org/bin/winnt/x86/0.6/julia-0.6-latest-win32.exe"
-  - JULIA_URL: "https://julialang-s3.julialang.org/bin/winnt/x64/0.6/julia-0.6-latest-win64.exe"
-  - JULIA_URL: "https://julialangnightlies-s3.julialang.org/bin/winnt/x86/julia-latest-win32.exe"
-  - JULIA_URL: "https://julialangnightlies-s3.julialang.org/bin/winnt/x64/julia-latest-win64.exe"
+  - julia_version: 0.6
```
Comment (quinnj) [inline appveyor.yml]: Ooooh....fancy.

File: src/DecFP.jl (hunk 1 above)
Comment (quinnj) [reply src/DecFP.jl]: It's there.

**Pull request merged**